video_xblock/tests/unit/mocks/wistia.py

```diff
@@ -38,208 +32,3 @@ def get(self):
         if self.event == 'not_authorized':
             self.return_value = ResponseStub(status_code=401)
         return lambda x: self.return_value
-
-
-class WistiaDefaultTranscriptsMock(BaseMock):
```

Comment:
Partially correct. The main reason is you can't mock every single inner utility in a tested method, because of the fact that backends may (and they do) perform that tested method each in its own way... So the "I" of ["F.I.R.S.T."](https://github.com/ghsukumar/SFDC_Best_Practices/wiki/F.I.R.S.T-Principles-of-Unit-Testing) is violated. I think...